lib/shared/store.js

```diff
@@ -120,6 +120,9 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         })
       }
 
+      // Convert object instances to plain objects
+      action.data = JSON.parse(JSON.stringify(action.data))
```

Comment:
Wouldn't something like `_.assign({}, action.data)` or `{ ...action.data }` work here?

Reply:
Good catch! `_.assign` seems to do the trick.

Reply:
`_.assign` didn't work as expected here?